### ISSUE TITLE
GIF to JXL slows speed down

### ISSUE THREAD

**Issue by Dry-Leaf** [libjxl/libjxl#1399]
For any gif I've tried larger than 1mb, after using cjxl to convert it to a jxl file, the speed noticeably drops. This is the case when using qView and Chromium(which is even worse). I've tried the file on windows and ubuntu and there's no difference.

**Comment by jonsneyers** [comment]
Does encoding with --faster_decoding=3 help? Or e.g. using lossy instead of lossless, say doing -e 6 -d 1.

Trying to figure out if this is the decoder being too slow or some kind of frame duration signaling issue...

**Comment by Dry-Leaf** [comment]
> Does encoding with --faster_decoding=3 help? Or e.g. using lossy instead of lossless, say doing -e 6 -d 1.
Neither of these help(on their own or together). The result is still much slower than the original. I don't know how relevant this is, but lossless conversion to webp doesn't have this issue, so I would assume my machine isn't the cause.

**Comment by jonsneyers** [comment]
Can you share an example gif for which this happens?

**Comment by Dry-Leaf** [comment]
![pen](https://user-images.githubusercontent.com/96496785/167313305-077d79d3-0818-4c96-bb42-d2a8c1f3d8d5.gif)
[pen.zip](https://github.com/libjxl/libjxl/files/8647678/pen.zip)

I tried with quite a few, and this is the best example I can post here. You can tell especially at the end with how much time it takes for the pen to fall. Fast movement and larger file sizes seem like the main factor.

**Comment by jonsneyers** [comment]
Yes, I can reproduce. Looks like the decoder is too slow for realtime, and there are too many frames to keep the whole loop in memory. So it ends up dropping lots of frames.

**Comment by ValZapod** [comment]
> Looks like the decoder is too slow for realtime,

Libgif?

P.S. You can just attach jxl files in github by drag and drop. Just rename it to .jpeg. it will work in chrome if flag is on.

**Comment by ValZapod** [comment]
There is a report that it happens with ffmpeg too, but ffmpeg does not use your native gif decoder... Strange, https://www.reddit.com/r/ffmpeg/comments/ukjii7/comment/i7qxaja/

**Comment by Dry-Leaf** [comment]
> There is a report that it happens with ffmpeg too, but ffmpeg does not use your native gif decoder... Strange, https://www.reddit.com/r/ffmpeg/comments/ukjii7/comment/i7qxaja/

I wrote that. ffmpeg can't convert gif to jxl directly. I only every used cjxl.
